packages/qa/utils/ApiRequests.ts

```diff
@@ -1,13 +1,13 @@
 import axios from 'axios'
 
 export async function getStudyEngagementInfo(studyId: string) {
-  const url = `https://test.cpmsapi.crncc.nihr.ac.uk/api/v1/studies/${studyId}/engagement-info`
+  const url = `${process.env.SE_TEST_API_URL}api/v1/studies/${studyId}/engagement-info`
   const username = process.env.SE_TEST_API_USERNAME
   const password = process.env.SE_TEST_API_PASSWORD
 
   // check if the environment variables are set
-  if (!username || !password) {
-    throw new Error('SE API username or password is not set in environment variables.')
+  if (!username || !password || !url) {
```

Comment:
I think we need to explicitly check that `process.env.SE_TEST_API_URL` is undefined - right now, if it was undefined, the `url` string will still be true i.e. it would be:

const url = `undefiendapi/v1/studies/${studyId}/engagement-info`

Reply:
great point - updated code